composer.json

```diff
@@ -16,9 +16,12 @@
     ],
     "require": {
         "composer/installers": "^2.0",
+        "drupal/component": "^1.0@RC",
```

Comment:
If I remember correctly we need only components module
If so, then component should be uninstalled